pizza-facts-&-trivia.md

```diff
@@ -19,6 +19,7 @@ Some say that the queen’s favorite pizza toppings were mozzarella, tomato, and
 10. Pizza was first developed as an easy and affordable meal for low-income families. Pizza may be an international favorite today but it has its origins as a last resort meal for low-income families in Naples.
 While pizza today is enjoyed by people across different classes of society, it was a meal reserved for low-income people who were trying to cut back on costs. It was an easy and cheap meal. Families would prepare the dough and top it off with cheese and tomatoes. According to researchers, people thought that tomatoes were poisonous during this period. This was why pizza was reserved for low-income families.
 11. The Hawaiian pizza, featuring pineapple and ham as its key toppings, was not invented in Hawaii. It was actually created in Canada. Sam Panopoulos, a Greek immigrant to Canada, is credited with making the first Hawaiian pizza in his restaurant, the Satellite Restaurant, in Chatham, Ontario, in 1962. This sweet and savory pizza combination has since become a popular and divisive choice for pizza lovers around the world.
+12. Legend has it that the Margherita pizza, a classic pizza topped with tomatoes, mozzarella cheese, basil, and a drizzle of olive oil, was created in honor of Queen Margherita of Savoy, the Queen of Italy in the late 19th century. The combination of red tomatoes, white mozzarella, and green basil was said to represent the colors of the Italian flag. Whether it's a fact or just a charming tale, the Margherita pizza remains a beloved and timeless classic around the world
```

Comment:
[suggestion] Interesting as this fact is, it was mentioned in number 5 already, so can you come up with a different one?
```suggestion
12. Legend has it that the Margherita pizza, a classic pizza topped with tomatoes, mozzarella cheese, basil, and a drizzle of olive oil, was created in honor of Queen Margherita of Savoy, the Queen of Italy in the late 19th century. The combination of red tomatoes, white mozzarella, and green basil was said to represent the colors of the Italian flag. Whether it's a fact or just a charming tale, the Margherita pizza remains a beloved and timeless classic around the world
```

Reply:
yes i can add another interesting fact ...
![DoThisGIF](https://github.com/open-sauced/pizza-verse/assets/84468625/1ddac0bc-2b25-4043-997c-87037d456f34)

Reply:
please have a look on the new commit i have submitted the unique pizza fact now